tutorials/39_Embedding_Metadata_for_Improved_Retrieval.ipynb

```diff
@@ -0,0 +1,431 @@
+    "\n",
+    "indexing_pipeline.run({\"cleaner\":{\"documents\": raw_docs}})\n",
+    "indexing_with_metadata_pipeline.run({\"cleaner\":{\"documents\": raw_docs}})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparing Retrieval With and Without Embedded Metadata\n",
+    "\n",
+    "As a final step, we will be creating a retrieval pipeline that will have 2 retrivers:\n",
+    "- First: retrieving from the `document_store`, where we have not embedded metadata.\n",
+    "- Second: retireving from the `document_store_with_embedded_metadata`, where we have embedded metadata.\n",
+    "\n",
+    "We will then be able to compare the restults and see if embedding metadata has helped with retrieval in this case.\n",
+    "\n",
+    "> 💡 Here, we are using the `InMemoryEmbeddintRetriever` because we used the `InMemoryDocumentStore` above. If you're using another documet store, change this to use the accompanying embedding retriever for the document store you are using. Check out the [Embedders Documentation](https://docs.haystack.deepset.ai/v2.0/docs/embedders) for a full list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<haystack.pipeline.Pipeline at 0x19e0933b0>"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from haystack.components.embedders import SentenceTransformersTextEmbedder\n",
+    "from haystack.components.retrievers.in_memory import InMemoryEmbeddingRetriever\n",
+    "\n",
+    "retrieval_pipeline = Pipeline()\n",
+    "retrieval_pipeline.add_component(\"text_embedder\", SentenceTransformersTextEmbedder(model=\"thenlper/gte-large\"))\n",
+    "retrieval_pipeline.add_component(\"retriever\", InMemoryEmbeddingRetriever(document_store=document_store, scale_score=False, top_k=3))\n",
+    "retrieval_pipeline.add_component(\"retriever_with_embeddings\", InMemoryEmbeddingRetriever(document_store=document_store_with_embedded_metadata, scale_score=False, top_k=3))\n",
+    "\n",
+    "retrieval_pipeline.connect(\"text_embedder\", \"retriever\")\n",
+    "retrieval_pipeline.connect(\"text_embedder\", \"retriever_with_embeddings\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's run the pipeline and compare the restuls from `retriever` and `retirever_with_embeddings`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Batches: 100%|██████████| 1/1 [00:00<00:00, 16.26it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Retriever Results:\n",
```

Comment:
it seems we get the same results with and without embedding metadata :slightly_smiling_face:

Reply:
@anakin87 these are the results I get:
<img width="1766" alt="image" src="https://github.com/deepset-ai/haystack-tutorials/assets/15802862/07520be1-b86d-4e1e-9339-c79b80ee7e99">

Reply:
The image is ok but different from the output in this PR.

Reply:
ohh that's probably because of a previous run I did and committed, those don't get pushed to the tutorials page so np